compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1398,7 +1517,7 @@ class Definitions {
         ScalaPackageClass.enter(m)
 
       // force initialization of every symbol that is synthesized or hijacked by the compiler
-      val forced = syntheticCoreClasses ++ syntheticCoreMethods ++ ScalaValueClasses()
+      val forced = syntheticCoreClasses(ctx) ++ syntheticCoreMethods ++ ScalaValueClasses()
```

Comment:
Same question as above: revert the change?

Reply:
Done